index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="aboutMe">
+            <h1 class="name">John Doe</h1>
+            <p class="jobDescription">Software Developer, United States</p>
+            <div class="bio">
+                I am an intern for DigitalNest. I come from a background where everything I ever did was outdoors and got curious as to how the pages and products we use every day worked. I am self thaught when it comes to web development but I did study for and have some experience with the software development part making some projects.
```

Comment:
Text shouldn't be within `<div></div>` tags they should always be within `<p></p>` or headings span etc ..